Packs/Sysdig/Integrations/Sysdig-Response-Actions/SysdigResponseActions.py

```diff
@@ -0,0 +1,313 @@
+    Returns:
+        The public API URL
+    """
+    match = re.search(r"https://(\w+)\.app\.sysdig\.com", base_url)
+    if match:
+        region = match.group(1)  # Extract the region
+        return f"https://api.{region}.sysdig.com"
+    raise ValueError(f"Invalid base URL format. Unable to extract region. Base URL should be in the format https://<region>.app.sysdig.com. Provided: {base_url}.")
+
+""" COMMAND FUNCTIONS """
+
+def execute_response_action_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    Call the Actions Response API
+    """
+    full_url = _get_public_api_url(client.base_url) + '/secure/response-actions/v1alpha1/action-executions'
+    data = _build_data_payload(args)
+
+    result = client.call_sysdig_api(method = 'POST', full_url = full_url, json_data = data)
+
+    return CommandResults(
+        outputs_prefix="execute_response_action.Output",
+        outputs=result
+    )
+
+def get_action_execution_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    '''
+    Get the status of an action execution
+    '''
+    action_execution_id = args.get("action_execution_id")
+    full_url = _get_public_api_url(client.base_url) + f'/secure/response-actions/v1alpha1/action-executions/{action_execution_id}'
+
+    result = client.call_sysdig_api(method = 'GET', full_url = full_url)
+
+    return CommandResults(
+        outputs_prefix="get_action_execution.Output",
+        outputs=result
+    )
+
+def create_system_capture_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    ''''
+    Trigger a sysdig system capture
+    '''
+    data = _build_capture_payload(args)
+    result = client.call_sysdig_api(method = 'POST', url_suffix = '/api/v1/captures', json_data = data)
+
+    return CommandResults(
+        outputs_prefix="create_system_capture.Output",
+        outputs=result
+    )
+
+def get_capture_file_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    Download a sysdig capture file (.scap). You must provide the capture ID. You can then use the scap file to analyze it with Stratoshark for example.
+    """
+
+    capture_id = args.get("capture_id") 
+    if not capture_id:
+        raise ValueError("capture_id is required")
+    url_suffix = f"/api/v1/captures/{capture_id}/download"
+
+    # The response is a binary file, so we set the resp_type to 'content'
+    result = client.call_sysdig_api(url_suffix = url_suffix, resp_type='content')
+    incident_id = demisto.incident().get('id')
+    file_name = f'{incident_id}_{capture_id}.scap'
+
+    # Save the file in the War Room
+    demisto.results(fileResult(file_name, result, EntryType.FILE))
+
+    readable_output = f"# Capture Taken\n**{file_name}** saved successfully to the War Room"
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="download_capture_file.Output",
```

Comment:
```suggestion
outputs_prefix="get_capture_file.Output"
outputs=?
```
Like in the YML and README files
What is the output (we can't set a outputs_prefix without an output)